src/client/component/filesystem.cpp

```diff
@@ -67,15 +67,18 @@ namespace filesystem
 			const auto code = game::SEH_GetCurrentLanguageName();
 
 			paths.push_back(path);
-			
-			if (is_fallback_lang())
-			{
-				paths.push_back(path / "fallback");
-			}
 
-			if (is_polrus_lang())
+			if (!::utils::io::file_exists("players2/default/disable_fallback_fonts"))
```

Comment:
Use a dvar

Reply:
[51c24e2](https://github.com/fedddddd/h2-mod/pull/400/commits/51c24e2a827b2085b1196565dc45514fd2d5f0e0) but, for some reason, it reads the default value instead of the user-defined one